### ISSUE TITLE
Does not recognize `payable`

### ISSUE THREAD

**Issue by kopy-kat** [defi-wonderland/natspec-smells#32]
Example:

```solidity
address payable constant SWAPROUTER_ADDRESS = payable(0xE592427A0AEce92De3Edee1F18E0157C05861564);
```

Error message:
```
CompileInferenceError: Failed parsing imports at [PATH/CONTRACT]:66:1 - Expected " ", "/*", "//", "[", "\t", "\x0B", "\x0C", "constant", " ", "﻿", [ , , , - , , ,　], or [\n,\r, , ] but "p" found.
    at findAllFiles (/.../node_modules/.pnpm/solc-typed-ast@17.0.3_typescript@5.3.3/node_modules/solc-typed-ast/dist/compile/inference/imports.js:128:23)
    at compileSol (/.../node_modules/.pnpm/solc-typed-ast@17.0.3_typescript@5.3.3/node_modules/solc-typed-ast/dist/compile/utils.js:160:44)
    at async getProjectCompiledSources (/.../node_modules/.pnpm/@defi-wonderland+natspec-smells@1.0.3_typescript@5.3.3/node_modules/@defi-wonderland/natspec-smells/lib/utils.js:29:27)
    at async /.../node_modules/.pnpm/@defi-wonderland+natspec-smells@1.0.3_typescript@5.3.3/node_modules/@defi-wonderland/natspec-smells/lib/main.js:16:25
```

**Comment by gas1cent** [comment]
@kopy-kat hey, thanks for the report! What solidity version are you using? Any chance I can see the full contract?

**Comment by gas1cent** [comment]
Perfect, thanks. It seems to be the parser issue, I've reported it here: https://github.com/Consensys/solc-typed-ast/issues/255
Will update you once we find a solution.

**Comment by gas1cent** [comment]
Hey @kopy-kat! Just fyi, the bug has been addressed by the Consensys team and we've included the fix in [v1.1.1](https://github.com/defi-wonderland/natspec-smells/releases/tag/v1.1.1). Give it a try :)
